llarp/router/router.cpp

```diff
@@ -50,6 +50,7 @@ namespace llarp
       , _dht(llarp_dht_context_new(this))
       , inbound_link_msg_parser(this)
       , _hiddenServiceContext(this)
+      , _randomStartDelay(1000 * (llarp::randint() % 30) + 10)
   {
```

Comment:
I've used a LERP utility function in the past for this sort of thing. Looks like `C++20` is adding LERP, mmmmm:

https://en.cppreference.com/w/cpp/numeric/lerp

Reply:
This looks like it's going to produce a value from {10, 1010, 2010, ..., 29010} to delay (in milliseconds).  Is this the intended logic?  The 10 (especially the initial one) seems a little strange to me.

Perhaps it was meant to be `1000 * ((llarp::randint() % 30) + 10)` to produce a random value from {10000, 11000, 12000, ..., 39000}?

But yeah, with Stephen's suggestion it could be: `10000 + llarp::randint() % 30000` to get something that makes a (nearly, and good enough) uniform random choice of all the integers between 10000 and 39999.

Reply:
i am unsure if milisecond accuracy is needed here in the randomization.